OpenQA/Qemu/DriveDevice.pm

```diff
@@ -111,7 +112,8 @@ sub gen_cmdline {
         if (defined $path->controller) {
             push(@params, 'bus=' . $path->controller->id . '.0');
         }
-        $self->_push_ifdef(\@params, 'bootindex=', $self->bootindex);
+	# Configure bootindex only for first path
+        $self->_push_ifdef(\@params, 'bootindex=', $self->bootindex) if ($current_path++ == 1);
```

Comment:
Is the first path always the bootable one? Do we know that `@paths` is always ordered that way?

Small note wrt coding style: code should only be indented with spaces.

Reply:
yes, i suppose  according to `my $paths = @{$self->paths} < 1 ? [OpenQA::Qemu::DrivePath->new()] : $self->paths;`, but @richiejp can comment it further

Reply:
You could check that `$path->id == '0'` instead so that the order of `@paths` is not significant (assuming it starts at zero, I can't remember).

I don't think it matters which path to a particular device is chosen as bootindex zero (so long as it is deterministic) unless we are testing QEMU itself.

Reply:
code updated to `$path->id eq 'path0'`, verification run in description updated as well

Reply:
extended condition with empty path-id  to fix travis failures for non-multipath scenarios